### PR TITLE
Feature/highlight keywords - PST-110

### PR DESCRIPTION
**Pull request by duaakelzi**

### CODE REVIEW

**Review by AlbinaMuzafarova — APPROVED** [review]
Looks good to me, thanks Duaa! 

We could probably skip highlighting the title in the preview but that's up to client's taste.

File: client/src/components/DetailedPatentView.vue [hunk 1]
```diff
@@ -130,6 +136,19 @@ export default defineComponent({
...
         },
+        highlightTitle(title: string, keywords: string[]) {
+            const pattern = new RegExp(`(${keywords.join('|')})`, 'gi');
+            return title.replace(pattern, (match) => {
+                return '<mark style="background-color:rgba(245, 255, 129, 1)">' + match + '</mark>';
+            });
+        },
+        highlightAbstract(abstract: string, keywords: string[]) {
+            const pattern = new RegExp(`(${keywords.join('|')})`, 'gi');
+            console.log(pattern);
```
Comment (samuelhuebner) [inline client/src/components/DetailedPatentView.vue:147]: This console.log could be removed

Comment (duaakelzi) [reply client/src/components/DetailedPatentView.vue:147]: ah sure that was for testing i forgot to remove

**Review by samuelhuebner — APPROVED** [review]

**Review by icicala — APPROVED** [review]
Looks cool! I like it very much! I have one question:
In case a keyword is contained in another word should be highlighted or not?
In the picture below You can see the case
image

**Pull request merged**